doc/index.md

```diff
@@ -1,21 +1,29 @@
 # Introduction
 
-lua-http is an performant, capable Hyper Text Transfer Protocol (HTTP) and WebSocket (WS) library for Lua 5.1, 5.2, 5.3 and LuaJIT. lua-http can be utilized as a Server or Client and includes the following features: 
+lua-http is an performant, capable Hyper Text Transfer Protocol (HTTP) and WebSocket (WS) library for Lua 5.1, 5.2, 5.3 and LuaJIT. The software supports x86, x64 and Arm based systems, as well as GNU/Linux, OSX, FreeBSD and others. lua-http can be utilized as a server or client and includes the following features: 
```

Comment:
> The software supports x86, x64 and Arm based systems

lua-http is pure lua code. It has no demands for architecture.

> as well as GNU/Linux, OSX, FreeBSD and others

where lua-http can run is mainly limited by where cqueues works (which is BSDs, Linux, OSX, Solaris): if you can port cqueues to it, lua-http should automatically work.